utils/appliance/__init__.py

```diff
@@ -19,29 +19,31 @@
 import requests
 import traceback
 
+from sentaku import ImplementationContext
 from utils.mgmt_system import RHEVMSystem
 from mgmtsystem.virtualcenter import VMWareSystem
 
-from cfme.common.vm import VM
 from fixtures import ui_coverage
 from fixtures.pytest_store import store
-from utils import api, conf, datafile, db, trackerbot, db_queries, ssh, ports
+from utils import api, conf, datafile, db, db_queries, ssh, ports
 from utils.datafile import load_data_file
 from utils.events import EventTool
 from utils.log import logger, create_sublogger, logger_wrap
+from utils.appliance.endpoints.ui import navigate, CFMENavigateStep
+from utils.navigate import NavigateToSibling
 from utils.net import net_check, resolve_hostname
 from utils.path import data_path, patches_path, scripts_path
-from utils.providers import get_mgmt, get_crud
 from utils.version import Version, get_stream, pick, LATEST
 from utils.signals import fire
 from utils.wait import wait_for
 from utils import clear_property_cache
 
+from .endpoints.ui import ViaUI
```

Comment:
i believe the name endpoint no longer captures what it is
but im not sure what name to pick